itou/prescribers/admin.py

```diff
@@ -203,19 +204,27 @@ def response_change(self, request, obj):
         * validating authorization
         """
         if "_authorization_action_refuse" in request.POST:
-            obj.is_authorized = False
-            obj.authorization_status = models.PrescriberOrganization.AuthorizationStatus.REFUSED
-            obj.authorization_updated_at = now()
-            obj.authorization_updated_by = request.user
-            obj.save()
-            obj.refused_prescriber_organization_email().send()
+            # Same checks in change_form template to display the button
+            if request.user.is_superuser or obj.has_pending_authorization() or obj.has_refused_authorization():
```

Comment:
J'ai un peu de mal à comprendre ceci : pourquoi voudrait-on refuser une habilitation déjà refusée ? Je suspecte ce `obj.has_refused_authorization()` d'être quelques lignes trop haut.

Reply:
Je me suis trompé en inversant la logique initiale `if not` en `if` au dernier moment.
La logique doit être :
- si super user, open bar
- si pending, les 2 actions sont autorisées
- si refusé, action validate possible.
Bref, cela correspond exactement à tes remarques, je corrige cette erreur par un rewrite du commit...
Et je te fais signe pour relecture finale.